.github/workflows/e2e_library_and_topic.yml

```diff
@@ -0,0 +1,54 @@
+name: End-to-End tests
```

Comment:
Why library in the name of the file? (I don't see any library-related test in this workflow

Reply:
My mistake, I've changed the file name.